configs/components/openssl-3.0.rb

```diff
@@ -136,7 +136,7 @@
   if settings[:use_legacy_openssl_algos]
     pkg.apply_patch 'resources/patches/openssl/openssl-3-activate-legacy-algos.patch'
   else
-    configure_flags << 'no-legacy' << 'no-md4' << 'no-des'
+    configure_flags << 'no-legacy' << 'no-md4'
```

Comment:
Can you add a `no-des` (commented out) like we're doing for `no-bf` so we know why it's not being disabled.

Reply:
Updated with info about pgcrypto and ntlm